pkg/operator/ceph/cluster/osd/labels.go

```diff
@@ -36,13 +36,19 @@ const (
 	OSDOverPVCLabelKey = "ceph.rook.io/pvc"
 	// TopologyLocationLabel is the crush location label added to OSD deployments
 	TopologyLocationLabel = "topology-location-%s"
+	// CephImageVersionLabel is the ceph image version label added to PVC
+	CephImageVersionLabelKey = "ceph.rook.io/CreatedCephImageVersion"
```

Comment:
It's not really the version, it could be any tag. But if it's a supported version, then it will of course be the version tag. If you really prefer the "version" in the name, I am ok with `originalCephImageVersion`
```suggestion
	cephImageTagLabelKey = "ceph.rook.io/originalCephImageTag"
```

Reply:
That's true. The label stores image name not just a image tag. 
And I prefer `Created` rather than `Original` because I intended the label to indicate the image at the point of the creation of OSD,
so I will change it to `CreatedCephImage` and `CreatedRookImage`